salt/_logging/handlers.py

```diff
@@ -116,6 +119,7 @@ def flush(self):
                 super().handle(record)
             finally:
                 self.__emitting = False
+        # This will raise a ValueError if the file handle has been closed.
         super().flush()
```

Comment:
I'm wondering why they haven't tried to catch this exception here, as in any case it should be ok just to hide it, no?

Reply:
I had the same thought, but it's a cherry-pick, so I used upstream's code...